build/build.ps1

```diff
@@ -39,6 +40,7 @@ function Print-Usage() {
     Write-Host "  -skipTests              Don't run tests"
     Write-Host "  -test                   Run tests. Ignores skipTests"
     Write-Host "  -bootstrapOnly          Don't run build again with bootstrapped MSBuild"
+    Write-Host "  -rebuildWithBootstrap   Run build again with bootstrapped MSBuild. Ignores bootstrapOnly"
```

Comment:
In my head, these should be

```
-noBootstrap
-bootstrap
```

That'd be quicker to type on the command line. But I'll adjust based on consensus.

Reply:
In my head `-bootstrap` means "build with the bootstrapped MSBuild". I totally get that that's not 100% logical.